modules/nw-metallb-installing-operator-cli.adoc

```diff
@@ -108,7 +108,7 @@ spec:
 EOF
 ----
 
-. Confirm the install plan is in the namespace:
+. Confirm the install plan is in the namespace (Installation of operator may take a few seconds):
```

Comment:
@michaelryanpeter, is this kind of Operator disclaimer used elsewhere in the docs?

Reply:
We don't use this type of disclaimer in the OLM docs. However, I have seen similar things in other places:
* https://docs.openshift.com/container-platform/4.14/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.html#distr-tracing-deploy-production_distr-tracing-jaeger-configuring (step four)
* https://docs.openshift.com/container-platform/4.14/logging/cluster-logging-upgrading.html#logging-upgrading-clo_cluster-logging-upgrading (Verification step one)